lib/src/base-styles/_nx-color-swatches.scss

```diff
@@ -87,23 +88,22 @@
   --nx-swatch-green-90: hsl(var(--nx-swatch-green-hs), 90%);
   --nx-swatch-green-95: hsl(var(--nx-swatch-green-hs), 95%);
 
-  // Lime
-  --nx-swatch-lime-hs:102, 86%;
-  --nx-swatch-lime-10: hsl(var(--nx-swatch-lime-hs), 10%);
-  --nx-swatch-lime-20: hsl(var(--nx-swatch-lime-hs), 20%);
-  --nx-swatch-lime-25: hsl(var(--nx-swatch-lime-hs), 25%);
-  --nx-swatch-lime-30: hsl(var(--nx-swatch-lime-hs), 30%);
-  --nx-swatch-lime-35: hsl(var(--nx-swatch-lime-hs), 35%);
-  --nx-swatch-lime-40: hsl(var(--nx-swatch-lime-hs), 40%);
-  --nx-swatch-lime-45: hsl(var(--nx-swatch-lime-hs), 45%);
-  --nx-swatch-lime-50: hsl(var(--nx-swatch-lime-hs), 50%);
-  --nx-swatch-lime-60: hsl(var(--nx-swatch-lime-hs), 60%);
-  --nx-swatch-lime-70: hsl(var(--nx-swatch-lime-hs), 70%);
-  --nx-swatch-lime-75: hsl(var(--nx-swatch-lime-hs), 75%);
-  --nx-swatch-lime-80: hsl(var(--nx-swatch-lime-hs), 80%);
-  --nx-swatch-lime-85: hsl(var(--nx-swatch-lime-hs), 85%);
-  --nx-swatch-lime-90: hsl(var(--nx-swatch-lime-hs), 90%);
-  --nx-swatch-lime-95: hsl(var(--nx-swatch-lime-hs), 95%);
+  // Lime - deprecated
```

Comment:
I realize that this is automated but it's hard to see the value in displaying these swatches, not to mention that the title still appears as just lime so the unwary might not realize that they are deprecated. Perhaps we should move all of these swatches elsewhere so they don't appear on this page.

Reply:
> Perhaps we should move all of these swatches elsewhere so they don't appear on this page.

Which source file they are in has no bearing on whether the automated logic will pick them up unfortunately.  If they are in the `gallery.css` output from the build, it will find them